stories/carousel/carousel.stories.js

```diff
@@ -477,3 +476,133 @@ carouselNoNavigation.parameters = {
+                        background-image: url(assets/images/backgrounds/Godafoss_waterfall_in_northern_Iceland.jpg);
+                        height: 15.5rem;
+                        background-size:cover;"
+                    role="img"
+                    aria-label="Carousel Image 5">
+                </div>
+            </div>
+
+            <div class="fd-carousel__item fd-carousel__item--active">
+                <div
+                    style="
+                        width:100%;
+                        background-image: url(assets/images/backgrounds/Godafoss_waterfall_in_northern_Iceland.jpg);
+                        height: 15.5rem;
+                        background-size:cover;"
+                    role="img"
+                    aria-label="Carousel Image 6">
+                </div>
+            </div>
+
+            <div class="fd-carousel__item fd-carousel__item--active">
+                <div
+                    style="
+                        width:100%;
+                        background-image: url(assets/images/backgrounds/Godafoss_waterfall_in_northern_Iceland.jpg);
+                        height: 15.5rem;
+                        background-size:cover;"
+                    role="img"
+                    aria-label="Carousel Image 7">
+                </div>
+            </div>
+        </div>
+        <div class="fd-carousel__page-indicator-container">
+            <button
+                class="fd-button fd-carousel__button fd-carousel__button--left"
+                data-slide="prev"
+                aria-label="Go to previous item">
+                <i class="sap-icon--slim-arrow-left"></i>
+            </button>
+            <ol class="fd-carousel__page-indicators">
+                <li
+                    data-slide-to="1"
+                    class="fd-carousel__page-indicator"></li>
+                <li
+                    data-slide-to="2"
+                    aria-label="Displaying item 2 of 3"
+                    class="fd-carousel__page-indicator fd-carousel__page-indicator--active"></li>
+                <li
+                    data-slide-to="3"
+                    class="fd-carousel__page-indicator"></li>
+            </ol>
+            <button
+                class="fd-button fd-carousel__button fd-carousel__button--right"
+                data-slide="next"
+                aria-label="Go to next item">
+                <i class="sap-icon--slim-arrow-right"></i>
+            </button>
+        </div>
+    </div>
+    <div style="display: none;" role="region" id="carousel-11" aria-live="polite">
+        Displaying item 2 of 3
+    </div>
+</div>
+`;
+
+horizontalCarousel.storyName = 'Items in horizontal direction';
+horizontalCarousel.parameters = {
+    docs: {
+        iframeHeight: 900,
+        storyDescription: `Carousel will have items which will spread horizontally and will be visible on navigation. For translation to work, all items should be in DOM.
```

Comment:
`translation` ?

Reply:
corrected the sentence.
For translate animation effect to work, all items should be in DOM.